docs/source/terms_and_model.rst

```diff
@@ -1169,8 +1044,12 @@ large) reference subsequences specified by a :ref:`SequenceLocation`.
 
 **Computational Definition**
 
-An expression of a sequence that is derived from a referenced
-sequence location.
+An approximate expression of a sequence that is derived from
```

Comment:
Suggested replacement:

A SequenceExpression that is derived from a referenced sequence location. The use of DerivedSequenceExpression suggests that the derived sequence is notional rather than exact, which is typically used for describing large regions for variation concepts where the precision of an exact sequence is unnecessary or undesirable.

Reply:
I might change the last part "unnecessary or undesireable." to "inconsequential." or maybe "not defining."?

Reply:
Done.